exporter/opentelemetry-exporter-otlp-proto-http/src/opentelemetry/exporter/otlp/proto/http/trace_exporter/__init__.py

```diff
@@ -65,7 +66,9 @@ def __init__(
     ):
         self._endpoint = endpoint or environ.get(
             OTEL_EXPORTER_OTLP_TRACES_ENDPOINT,
-            environ.get(OTEL_EXPORTER_OTLP_ENDPOINT, DEFAULT_ENDPOINT),
+            _append_trace_path(
+                environ.get(OTEL_EXPORTER_OTLP_ENDPOINT, DEFAULT_ENDPOINT)
```

Comment:
@srikanthccv 
Do we need to add this for [logs](https://github.com/open-telemetry/opentelemetry-python/pull/2462) as well? Probably in the future since it is not explicitly speced out right?

Reply:
yes, although I don't expect any difference. I think it is good to make that change in the #2462 itself.

Reply:
My guess is it is needed for metrics as well?

Reply:
Yes, but we don't yet have http metrics exporter.